contributed_definitions/NXwaveplate.nxdl.xml

```diff
@@ -83,6 +83,11 @@
             <dim index="1" value="N_wavelengths"/>
         </dimensions>
     </field>
+    <group name="retardance_distribution" type="NXdata">
```

Comment:
https://en.wikipedia.org/wiki/Waveplate then the docstring should use the term "retardance" instead of the typoed retardence and also 

year name needs to be reverted to 2014-2024